src/components/Leftside.jsx

```diff
@@ -23,7 +23,7 @@ const Leftside = (props) => {
       <ArtCard>
         <UserInfo>
           <CardBackground />
-          <Link to='/profile'>
+          <Link to='/profile' className='welcome'>
```

Comment:
and here I have  ... '/profile'.  I was not sure which way was right but I found a site that told me this is the correct way. What is your opinion? <Profile/> or  '/profile'